IABot/Parser/parse.php

```diff
@@ -34,6 +34,9 @@
 * @license https://www.gnu.org/licenses/gpl.txt
 * @copyright Copyright (c) 2016, Maximilian Doerr
 */
+require __DIR__ . '/../../vendor/autoload.php';
```

Comment:
This is required by composer to load CheckIfDead class. I *suspect* we can do away with this line keeping only the 'use Wikimedia\DeadlinkChecker\CheckIfDead;' statement, but someone needs to test it.